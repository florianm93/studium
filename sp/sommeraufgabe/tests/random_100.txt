*** Kommentar 1
*** Kommentar 2
*** Kommentar 3
349 81 76
331 333 27
246 305 31
183 176 35
5 127 76
297 96 67
129 51 15
279 29 64
192 36 35
249 21 53
213 384 92
221 106 84
287 345 44
106 213 63
41 85 91
65 197 19
77 43 73
72 59 17
301 380 36
187 290 65
34 164 52
260 329 34
301 394 66
63 26 39
22 31 25
385 114 96
398 334 95
241 101 19
198 221 21
80 251 63
80 42 26
123 335 79
119 342 53
252 59 28
60 21 95
198 248 27
47 17 2
33 347 26
266 379 55
31 136 7
7 226 3
330 97 23
66 84 96
314 24 11
389 77 48
308 341 37
392 146 73
259 105 57
382 342 75
390 139 78
183 171 56
32 273 17
312 159 21
161 220 5
206 216 60
53 365 9
288 167 30
51 33 29
63 183 24
119 36 41
371 141 87
36 89 43
230 216 85
378 344 7
230 7 24
19 275 78
88 166 11
6 245 91
377 102 23
24 266 44
21 125 16
310 282 13
19 281 13
288 76 88
183 333 33
169 144 96
5 102 6
55 38 88
38 269 17
387 294 94
317 104 8
301 339 56
164 111 22
376 65 95
165 384 87
129 39 97
292 135 99
54 42 79
295 309 42
357 332 50
156 252 7
224 216 10
243 373 12
286 124 14
141 291 73
7 344 84
287 159 13
318 270 12
118 132 45
142 231 50
