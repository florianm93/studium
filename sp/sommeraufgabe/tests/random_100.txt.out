**
** Daten aus Datei: tests/random_100.txt
**

Senderpositionen (X,Y) und Senderradien:
S01: 349.000,  81.000,  76.000
S02: 331.000, 333.000,  27.000
S03: 246.000, 305.000,  31.000
S04: 183.000, 176.000,  35.000
S05:   5.000, 127.000,  76.000
S06: 297.000,  96.000,  67.000
S07: 129.000,  51.000,  15.000
S08: 279.000,  29.000,  64.000
S09: 192.000,  36.000,  35.000
S10: 249.000,  21.000,  53.000
S11: 213.000, 384.000,  92.000
S12: 221.000, 106.000,  84.000
S13: 287.000, 345.000,  44.000
S14: 106.000, 213.000,  63.000
S15:  41.000,  85.000,  91.000
S16:  65.000, 197.000,  19.000
S17:  77.000,  43.000,  73.000
S18:  72.000,  59.000,  17.000
S19: 301.000, 380.000,  36.000
S20: 187.000, 290.000,  65.000
S21:  34.000, 164.000,  52.000
S22: 260.000, 329.000,  34.000
S23: 301.000, 394.000,  66.000
S24:  63.000,  26.000,  39.000
S25:  22.000,  31.000,  25.000
S26: 385.000, 114.000,  96.000
S27: 398.000, 334.000,  95.000
S28: 241.000, 101.000,  19.000
S29: 198.000, 221.000,  21.000
S30:  80.000, 251.000,  63.000
S31:  80.000,  42.000,  26.000
S32: 123.000, 335.000,  79.000
S33: 119.000, 342.000,  53.000
S34: 252.000,  59.000,  28.000
S35:  60.000,  21.000,  95.000
S36: 198.000, 248.000,  27.000
S37:  47.000,  17.000,   2.000
S38:  33.000, 347.000,  26.000
S39: 266.000, 379.000,  55.000
S40:  31.000, 136.000,   7.000
S41:   7.000, 226.000,   3.000
S42: 330.000,  97.000,  23.000
S43:  66.000,  84.000,  96.000
S44: 314.000,  24.000,  11.000
S45: 389.000,  77.000,  48.000
S46: 308.000, 341.000,  37.000
S47: 392.000, 146.000,  73.000
S48: 259.000, 105.000,  57.000
S49: 382.000, 342.000,  75.000
S50: 390.000, 139.000,  78.000
S51: 183.000, 171.000,  56.000
S52:  32.000, 273.000,  17.000
S53: 312.000, 159.000,  21.000
S54: 161.000, 220.000,   5.000
S55: 206.000, 216.000,  60.000
S56:  53.000, 365.000,   9.000
S57: 288.000, 167.000,  30.000
S58:  51.000,  33.000,  29.000
S59:  63.000, 183.000,  24.000
S60: 119.000,  36.000,  41.000
S61: 371.000, 141.000,  87.000
S62:  36.000,  89.000,  43.000
S63: 230.000, 216.000,  85.000
S64: 378.000, 344.000,   7.000
S65: 230.000,   7.000,  24.000
S66:  19.000, 275.000,  78.000
S67:  88.000, 166.000,  11.000
S68:   6.000, 245.000,  91.000
S69: 377.000, 102.000,  23.000
S70:  24.000, 266.000,  44.000
S71:  21.000, 125.000,  16.000
S72: 310.000, 282.000,  13.000
S73:  19.000, 281.000,  13.000
S74: 288.000,  76.000,  88.000
S75: 183.000, 333.000,  33.000
S76: 169.000, 144.000,  96.000
S77:   5.000, 102.000,   6.000
S78:  55.000,  38.000,  88.000
S79:  38.000, 269.000,  17.000
S80: 387.000, 294.000,  94.000
S81: 317.000, 104.000,   8.000
S82: 301.000, 339.000,  56.000
S83: 164.000, 111.000,  22.000
S84: 376.000,  65.000,  95.000
S85: 165.000, 384.000,  87.000
S86: 129.000,  39.000,  97.000
S87: 292.000, 135.000,  99.000
S88:  54.000,  42.000,  79.000
S89: 295.000, 309.000,  42.000
S90: 357.000, 332.000,  50.000
S91: 156.000, 252.000,   7.000
S92: 224.000, 216.000,  10.000
S93: 243.000, 373.000,  12.000
S94: 286.000, 124.000,  14.000
S95: 141.000, 291.000,  73.000
S96:   7.000, 344.000,  84.000
S97: 287.000, 159.000,  13.000
S98: 318.000, 270.000,  12.000
S99: 118.000, 132.000,  45.000
S100: 142.000, 231.000,  50.000

Frequenzzuordnung:
S76->1
S43->2
S87->2
S04->3
S86->3
S74->4
S08->5
S63->5
S12->6
S15->4
S06->3
S14->6
S48->7
S88->5
S78->6
S01->1
S61->8
S26->9
S35->7
S95->2
S51->8
S80->1
S57->10
S68->1
S20->3
S84->10
S21->3
S17->8
S66->4
S62->1
S30->5
S11->1
S39->2
S89->3
S99->9
S32->7
S13->4
S82->5
S58->9
S60->10
S100->4
S55->7
S23->6
S50->5
S18->11
S24->12
S27->7
S31->13
S96->3
S85->5
S10->8
S19->8
S46->9
S53->6
S42->6
S47->7
S22->7
S90->10
S49->11
S25->10
S59->7
S34->9
S02->12
S45->11
S70->2
S29->6
S94->5
S97->1
S81->11
S69->6
S37->1
S33->6
S07->9
S03->6
S71->7
S16->8
S09->2
S36->9
S28->8
S77->3
S73->6
S52->7
S79->8
S67->5
S54->3
S40->5
S91->1
S83->4
S75->4
S92->3
S65->1
S93->3
S38->2
S72->2
S44->2
S98->4
S64->2
S41->3
S56->1
Anzahl benoetiger Frequenzen: 13
 Frequenz | Sender
=======================================================================================
    0     |  S05  
    1     |  S01  S11  S37  S56  S62  S65  S68  S76  S80  S91  S97  
    2     |  S09  S38  S39  S43  S44  S64  S70  S72  S87  S95  
    3     |  S04  S06  S20  S21  S41  S54  S77  S86  S89  S92  S93  S96  
    4     |  S13  S15  S66  S74  S75  S83  S98  S100  
    5     |  S08  S30  S40  S50  S63  S67  S82  S85  S88  S94  
    6     |  S03  S12  S14  S23  S29  S33  S42  S53  S69  S73  S78  
    7     |  S22  S27  S32  S35  S47  S48  S52  S55  S59  S71  
    8     |  S10  S16  S17  S19  S28  S51  S61  S79  
    9     |  S07  S26  S34  S36  S46  S58  S99  
    10     |  S25  S57  S60  S84  S90  
    11     |  S18  S45  S49  S81  
    12     |  S02  S24  
    13     |  S31  
